Lib/test/test_os.py

```diff
@@ -1517,7 +1528,8 @@ def mock_execve(name, *args):
         os.execve = orig_execve
         os.defpath = orig_defpath
 
-
+@unittest.skipUnless(hasattr(os, 'execv'),
+                         "No os.execv or os.execve function to test.")
```

Comment:
nitpick: please align the string to the start of hasattr.

Reply:
Done.